contribs/gmf/src/directives/authenticationdirective.js

```diff
@@ -0,0 +1,247 @@
+goog.provide('gmf.AuthenticationController');
+goog.provide('gmf.authenticationDirective');
+
+goog.require('gmf');
+goog.require('gmf.Authentication');
+goog.require('ngeo.modalDirective');
+
+
+gmfModule.value('gmfAuthenticationTemplateUrl',
+    /**
+     * @param {angular.JQLite} element Element.
+     * @param {angular.Attributes} attrs Attributes.
+     */
+    function(element, attrs) {
+      var templateUrl = attrs['gmfAuthenticationTemplateurl'];
+      return templateUrl !== undefined ? templateUrl :
+          gmf.baseTemplateUrl + '/authentication.html';
+    });
+
+
+/**
+ * An "authentication" directive for a GeoMapFish application.
+ *
```

Comment:
Could you add some explanation about the content of the directive.
Just that it is a form with options blabla